src/Files.App/UserControls/NavigationToolbar.xaml

```diff
@@ -267,13 +267,13 @@
 					x:Name="Back"
 					AccessKey="B"
 					AccessKeyInvoked="Button_AccessKeyInvoked"
-					AutomationProperties.FullDescription="{x:Bind ViewModel.Commands.NavigateBack.Description, Mode=OneWay}"
-					AutomationProperties.Name="{x:Bind ViewModel.Commands.NavigateBack.Label, Mode=OneWay}"
-					Command="{x:Bind ViewModel.Commands.NavigateBack, Mode=OneWay}"
-					IsEnabled="{x:Bind ViewModel.Commands.NavigateBack.IsExecutable, Mode=OneWay}"
+					AutomationProperties.FullDescription="{x:Bind Commands.NavigateBack.Description, Mode=OneWay}"
+					AutomationProperties.Name="{x:Bind Commands.NavigateBack.Label, Mode=OneWay}"
+					Command="{x:Bind Commands.NavigateBack, Mode=OneWay}"
+					IsEnabled="{x:Bind Commands.NavigateBack.IsExecutable, Mode=OneWay}"
 					Style="{StaticResource AddressToolbarButtonStyle}"
-					ToolTipService.ToolTip="{x:Bind ViewModel.Commands.NavigateBack.LabelWithHotKey, Mode=OneWay}">
-					<FontIcon FontSize="14" Glyph="{x:Bind ViewModel.Commands.NavigateBack.Glyph.BaseGlyph, Mode=OneWay}" />
+					ToolTipService.ToolTip="{x:Bind Commands.NavigateBack.LabelWithHotKey, Mode=OneWay}">
+					<FontIcon FontSize="14" Glyph="{x:Bind Commands.NavigateBack.Glyph.BaseGlyph}" />
```

Comment:
This can also be OneTime. The truth is the default is `OneTime`, but we prefer to define it for clarity.